releases/note-source/v14_0.rst

```diff
@@ -0,0 +1,373 @@
+
+Deterministic package version checking
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The codebase keeps track of which software versions were used to calculate a particular set of outputs as a means of tracking data provenance.
+However, in some circumstances this list was being stored inconsistently, leading to version mismatches being reported where non existed.
+This has now been resolved.
+
+:jirab:`DM-9439`
+
+.. _release-v14-0-cmodel-prior:
+
+Correct CModel priors
+^^^^^^^^^^^^^^^^^^^^^
+
+CModel priors are now weighted correctly relative to the likelihood.
+
+:jirab:`DM-9795`
+
+.. _release-v14-0-blended-debias:
+
+Correct blendedness debiasing calculation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Resolved a mathematical error.
+
+:jirab:`DM-10237`
+
+.. _release-v14-0-temp-local-bg:
+
+Fixed order of operations when using temporary local backgrounds in detection
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Resolved an issue which could result in missed isolated-object detections and incorrect ``Footprint``\s for large objects.
+
+:jirab:`DM-10271`
+
+.. _release-v14-0-peak-cull:
+
+Correct number of bands used for peak culling
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+During multi-band coadd processing, peaks may be rejected (or “culled”) if they have been detected in insufficiently many bands.
+A book-keeping error led to the number of bands in which a peak had been detected to be recorded incorrectly, and hence to the wrong peaks being culled.
+This has now been corrected.
+
+:jirab:`DM-11625`
+
+.. _release-v14-0-internal-improvements:
+
+Build and Code Improvements
+---------------------------
+
+- :ref:`release-v14-0-yaml-policy`
+- :ref:`release-v14-0-py3`
+- :ref:`release-v14-0-flaghandler`
+- :ref:`release-v14-0-numpy`
+- :ref:`release-v14-0-auto-bt`
+
+.. _release-v14-0-yaml-policy:
+
+YAML-based policy
+^^^^^^^^^^^^^^^^^
+
+The “policy” files associated with standard stack cameras, which define where and how datasets are persisted by the Butler, were updated to use a new `YAML <http://yaml.org>`_-based syntax.
+
+:jirab:`DM-7363`
+
+.. _release-v14-0-py3:
+
+Full support for Python 3.5
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+All of the Science Pipelines code included in this release is tested with both Python 2.7 and Python 3.5.
+It should also support Python 3.6, although that has not been rigorously tested.
+
+:jirab:`DM-7756`
+
+.. _release-v14-0-flaghandler:
+
+More Robust Handling of Flagged Measurements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``FlagHandler`` mechanism for setting flags indicating the status of particular measurements was enhanced to improve its robustness and ensure that flags are always set consistently.
+
+:jirab:`DM-9249`
+
+.. _release-v14-0-numpy:
+
+Support for recent NumPy
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Recent versions of NumPy have introduced a number of changes which were incompatible with usage in the Science Pipelines code.
```

Comment:
Do we recall what version of NumPy this is, specifically?

Reply:
1.13 needed some fixes.

Reply:
Both 1.12 (DM-9316) and 1.13 (DM-10926) needed some fixes, in fact.